geopandas/geodataframe.py

```diff
@@ -1533,6 +1519,24 @@ def __sub__(self, other):
         )
         return self.geometry.difference(other)
 
+    if compat.PANDAS_GE_025:
+        from pandas.core.accessor import CachedAccessor
+
+        plot = CachedAccessor("plot", geopandas.plotting.GeoplotAccessor)
```

Comment:
We should probably check with pandas if it's OK to use this CachedAccessor (it's not officially public API, `pandas.core` is considered private). But the problem with the public API `pd.api.extensions.register_dataframe_accessor` is that it will give a warning because we are overriding an existing name ..

Reply:
How do we go about this 😅

Reply:
I will check with pandas about this, but for now this is fine